### PR TITLE
Fixed issue that prevented to enable access plan `forced_redirect` property

### LINKED ISSUE #271: Cannot enable access plan `forced_redirect` property
Probably related to #267 

Request to `/access-plan/$id`
```json
{"redirect_forced":true}
```

Results in:
```json
{"code":"invalid_meta","message":"Cannot insert\/update the checkout_redirect_forced meta","data":null}
```

But disabling works:
```json
{"redirect_forced":false}
```

### PR DESCRIPTION
**Pull request by eri-trabiccolo**
## Description
Fixes #271

## How has this been tested?
manually

## Types of changes
Bug fix (non-breaking change which fixes an issue)

## Checklist:
- [x] My code has been tested.
- [x] My code passes all existing automated tests. 
- [x] My code follows the LifterLMS Coding & Documentation Standards.

### CODE REVIEW

File: .changelogs/issue_271.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+significance: patch
+type: fixed
+links:
+  - "#271"
+entry: Fixed issue that prevented to enable access plan `forced_redirect` property.
```
Comment (thomasplevy) [inline .changelogs/issue_271.yml]:
```suggestion
entry: Fixed issue that prevented updating the access plan `forced_redirect` property.
```

File: includes/server/class-llms-rest-access-plans-controller.php [hunk 2]
```diff
@@ -439,6 +439,7 @@ protected function prepare_item_for_database( $request ) {
 	 *
 	 * @since 1.0.0-beta.18
 	 * @since [version] Fixed reference to a non-existent schema property: visibiliy in place of visibility.
+	 *                      Fixed issue which prevented to enable redirect_forced property.
```
Comment (thomasplevy) [inline includes/server/class-llms-rest-access-plans-controller.php]:
```suggestion
	 *                      Fixed issue that prevented updating the access plan `forced_redirect` property.
```

**Review by thomasplevy — DISMISSED** [review]
Approved with language updates

File: includes/server/class-llms-rest-access-plans-controller.php (hunk 2 above)
Comment (eri-trabiccolo) [inline includes/server/class-llms-rest-access-plans-controller.php]:
```suggestion
	 *                      Fixed issue that prevented updating the access plan `redirect_forced` property.
```

**Review by eri-trabiccolo — COMMENTED** [review]
cool

File: .changelogs/issue_271.yml (hunk 1 above)
Comment (eri-trabiccolo) [inline .changelogs/issue_271.yml]:
```suggestion
entry: Fixed issue that prevented updating the access plan `redirect_forced` property.
```

**Review by thomasplevy — APPROVED** [review]

**Pull request merged**